tests/hubapi/demohub/biothing_studio/bin/hub.py

```diff
@@ -24,15 +23,9 @@
 logging = config.logger
 
 hasgene = HasGeneMapper(name="has_gene")
-pbuilder = partial(TaxonomyDataBuilder,mappers=[hasgene])
 server = HubServer(
     hub.dataload.sources,
     name="BioThings Studio",
-    managers_custom_args={
-        "build": {
-            "builder_class": pbuilder
-        }
-    }
```

Comment:
@remoteeng00  why `pbuilder` was removed here?

Reply:
yeah, I shoud not be removed. I reverted this change.